src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/building/internals.md.https://powershell.myget.org/feed/powershell-core/package/nuget/libpsl should work
```powershell
retry of URL failed with error: An error occurred while sending the request.
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```